sdk/vaa/structs.go

```diff
@@ -438,6 +438,37 @@ func (v *VAA) VerifySignatures(addresses []common.Address) bool {
 
 	return true
 }
+func (v *VAA) Verify(addresses []common.Address) bool {
```

Comment:
The only problem is that if you return `bool` instead of `error`, we can't log or show what the issue was with the `vaa`, which may be ok, I don't know if it's necessary to be so explicit.

Reply:
Now, returning err